File: readropslibrary/src/main/java/com/readrops/readropslibrary/localfeed/json/JSONAuthor.java
package com.readrops.readropslibrary.localfeed.json;

import com.google.gson.annotations.SerializedName;

public class JSONAuthor {

    private String name;

    private String url;

    @SerializedName("avatar")
    private String avatarUrl;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getAvatarUrl() {
        return avatarUrl;
    }

    public void setAvatarUrl(String avatarUrl) {
        this.avatarUrl = avatarUrl;
    }


}

File: readropslibrary/src/main/java/com/readrops/readropslibrary/localfeed/json/JSONAuthor.kt
package com.readrops.readropslibrary.localfeed.json

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class JSONAuthor(val name: String,
                      val url: String,
                      @Json(name = "avatar") val avatarUrl: String?)